examples/existing_vpc/main.tf

```diff
@@ -16,5 +16,6 @@ module "slz_vpc" {
   create_vpc          = false
   existing_vpc_id     = var.vpc_id
   create_subnets      = false
+  public_gateway_name = var.public_gateway_name
```

Comment:
hard code a name here instead of using variable value (perhaps use the var.prefix in the name)

Reply:
There is no var.prefix in this example. In the tests, I am passing like `fmt.Sprintf("%s-public-gateway", prefix),`